### PR TITLE
GROK-15160: Upgrade database version

### PR DESCRIPTION
**Pull request by VHlushchen**

### CODE REVIEW

File: aws/db.tf [hunk 1]
```diff
@@ -80,3 +80,17 @@ module "db" {
 
   tags = local.tags
 }
+
+data "aws_route53_zone" "private_zone" {
+  name         = var.private_zone_name
+  private_zone = true
+}
+
+resource "aws_route53_record" "www" {
```
Comment (dnillovna) [inline aws/db.tf:89]: let's rename it to something meaningful

File: aws/db.tf [hunk 2]
```diff
@@ -80,3 +80,17 @@ module "db" {
 
   tags = local.tags
 }
+
+data "aws_route53_zone" "private_zone" {
```
Comment (dnillovna) [inline aws/db.tf:84]: No need to data the resource. We create it inside the module.
https://github.com/datagrok-ai/tf-module-datagrok-core/blob/19fcab1827b86a67811f73c81c90e53372cf68d5/aws/lb.tf#L222-L229

File: aws/monitoring.tf [hunk 3]
```diff
@@ -13,7 +13,7 @@ resource "aws_sns_topic_subscription" "email" {
     compact(
       concat(
         var.monitoring.email_alerts_datagrok ?
-        ["monitoring@datagrok.ai", "spodolskaya@datagrok.ai", "dnahovskyi@datagrok.ai"] :
+        ["monitoring@datagrok.ai", "spodolskaya@datagrok.ai", "vhlushchenko@datagrok.ai"] :
```
Comment (dnillovna) [inline aws/monitoring.tf:16]: Remove monitoring@datagrok.ai, it is useless, unfortunately, due to AWS email spam

File: aws/variables.tf [hunk 4]
```diff
@@ -171,7 +171,11 @@ variable "rds_dg_password" {
   nullable    = true
   description = "The password for datagrok user in RDS. If it is not specified, the random password will be generated, 16 symbols long without special characters."
 }
-
+variable "private_zone_name" {
+  type = string
+  default = "datagrok.datagrok.public.internal"
+  description = "Private hosted_zone nane"
+}
```
Comment (dnillovna) [inline aws/variables.tf:178]: No need in the extra variable

**Review by dnillovna — CHANGES_REQUESTED** [review]

File: aws/monitoring.tf (hunk 3 above)
Comment (dnillovna) [reply aws/monitoring.tf:16]: @VHlushchen

File: aws/db.tf [hunk 5]
```diff
@@ -80,3 +80,12 @@ module "db" {
 
   tags = local.tags
 }
+
+resource "aws_route53_record" "db_private_dns" {
+  zone_id = aws_route53_zone.internal[0].zone_id
```
Comment (dnillovna) [inline aws/db.tf:85]: There is a condition for internal zone, so it is important to check it here too

File: aws/monitoring.tf (hunk 3 above)
Comment (VHlushchen) [reply aws/monitoring.tf]: Removed

File: aws/db.tf (hunk 5 above)
Comment (VHlushchen) [reply aws/db.tf:86]: Condition added

File: aws/db.tf [hunk 6]
```diff
@@ -80,3 +80,13 @@ module "db" {
 
   tags = local.tags
 }
+
+resource "aws_route53_record" "db_private_dns" {
+  count = var.create_route53_internal_zone ? 1 : 0
+  zone_id = aws_route53_zone.internal[0].zone_id
+  name    = "public_db.${aws_route53_zone.internal[0].name}"
+  type    = "CNAME"
```
Comment (dnillovna) [inline aws/db.tf:88]: Is it possible to use an alias instead of CNAME, like for LBs?

**Pull request merged**